### DISCUSSION TITLE
Lua: Include metadata YAML in pandoc.write output

### DISCUSSION THREAD

**Discussion by bpj** [jgm/pandoc#10175] [Q&A] [answered]
Is there a way to get the metadata included as a YAML block in `pandoc.write` output?

pandoc 3.4

**Comment by jgm** [comment]
Try setting `template` in writer options.  If `template` is empty, pandoc assumes a fragment is wanted and won't include the metadata.  Setting it to `$body$` should even be enough, if I recall correctly.

**Comment by bpj** [comment, marked as answer]
@jgm almost right: `{ template = '$titleblock$' }` in the writer options did the trick. I found this after peeking in the default template.

While I understand why this variable is named so for historical reasons it is somewhat misleading.
